picard.wdl

```diff
@@ -313,6 +313,49 @@ task CollectTargetedPcrMetrics {
     }
 }
 
+task CreateSequenceDictionary {
+    input {
+        File inputFile
+        String outputDir
+        String basenameInputFile = basename(inputFile)
+
+        String memory = "2G"
+        String javaXmx = "2G"
+        String dockerImage = "quay.io/biocontainers/picard:2.22.3--0"
+    }
+
+    command {
+        set -e
+        mkdir -p "$(dirname ~{outputDir})"
+        picard -Xmx~{javaXmx} \
```

Comment:
Also -XX parallel GC threads something to 1.
This prevents the garbage collection from spinning out of control on nodes with lots of memory. Sorry I did not spot it before!

Reply:
Should we add this to the other picard tasks as well?

Reply:
The time_minutes brach is also adding this to a lot of the tasks, so maybe only add it to new tasks here, otherwise the eventual merging will involve a lot of conflict resolving

Reply:
This task is new. So it can be added I think.

Reply:
Alright!

Reply:
yes, this was in response to @JasperBoom's question if it should be added to the other tasks as well.